tests/series_only/value_counts_test.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any
+
+import numpy as np
+import pytest
+
+import narwhals.stable.v1 as nw
+
+data = {"a": [4, 4, 6]}
+
+
+def test_value_counts(request: Any, constructor: Any) -> None:
+    if "pandas_series_nullable" in str(constructor):  # fails for py3.8
+        request.applymarker(pytest.mark.xfail)
+
+    s = nw.from_native(constructor(data), eager_only=True)["a"]
+
+    sorted_result = s.value_counts(sort=True)
+    assert sorted_result.columns == ["a", "count"]
+
+    expected = np.array([[4, 2], [6, 1]])
+    assert (sorted_result.to_numpy() == expected).all()
+
+    unsorted_result = s.value_counts(sort=False)
+    assert unsorted_result.columns == ["a", "count"]
+
+    a = unsorted_result.to_numpy()
+    assert (a[a[:, 0].argsort()] == expected).all()
```

Comment:
I think this is an old test, seeing as we're updating things, can we write it using `compare_dict`?

So
```python
    sorted_result = series.value_counts(sort=True)
    expected = {'b': [4, 6], 'count': [2, 1]}
    compare_dicts(sorted_result, expected)

    unsorted_result = series.value_counts(sort=False).sort('count', descending=True)
    compare_dicts(unsorted_result, expected)
```

for `sort=False`, the output is meant to be random, so I'm not sure we can make a fully deterministic test case - but we can at least check that if we then sort outselves, the result is correct

Reply:
yass! maybe I'll also extend the test a bit by adding more values, so it also checks the sort arg properly

Reply:
should it be  if "pandas_series_nullable" in str(constructor) **or** sys.version_info < (3,9 ):  OR **and?**